### ISSUE TITLE
WP Editors sought

### ISSUE THREAD

**Issue by dscotese** [voluntaryist/site#8]
I'd like to copy the content of each article on the site into WordPress.  This is a big job, obviously, but it can be done piece by piece.  In order to avoid two people trying to do the same thing, I propose that we use WP features like so:
- If you're willing to do some of this work, let me know and I'll make you an editor.  You will then be able to log into WP to do the other steps.
- Choose one of the items from the left menu.
- Make a category for your work, and name and describe it as you will see I did under "Posts | Categories" for "Fundamentals".
- Create the post(s) and add issues if you have any questions.

**Comment by Shepardh1** [comment]
I must be a real friggin moron... I have not had time to read the many emails that have been coming in, but I think in another week or two I will slow down enough to sift through them... sorry for my delay...

**Comment by Shepardh1** [comment]
I will take, "How I became a Voluntaryist"

:)

Was the format of my last entry ok?

|

*Shepard Humphries' Vision <http://shepardhumphries.com/vision.html> |
Owner |  **www.ShootInJH.com* <http://www.shootinjh.com/>
*Jackson Hole Shooting Experience  |  **Proudly the international leader in
luxury entertainment shooting.*
*    <http://www.linkedin.com/pub/shepard-humphries/14/53/80b>*

<https://www.tripadvisor.com/UserReviewEdit-g1798760-d1909756-e__2F__Attraction__5F__Review__2D__g1798760__2D__d1909756__2D__Reviews__2D__Jackson__5F__Hole__5F__Shooting__5F__Experience__2D__Jackson__5F__Hole__5F__Wyoming__2E__html-Jackson_Hole_Shooting_Experience-Jackson_Hole_Wyoming.html>

<https://www.tripadvisor.com/UserReviewEdit-g1798760-d1909756-e__2F__Attraction__5F__Review__2D__g1798760__2D__d1909756__2D__Reviews__2D__Jackson__5F__Hole__5F__Shooting__5F__Experience__2D__Jackson__5F__Hole__5F__Wyoming__2E__html-Jackson_Hole_Shooting_Experience-Jackson_Hole_Wyoming.html>

<https://www.tripadvisor.com/UserReviewEdit-g1798760-d1909756-e__2F__Attraction__5F__Review__2D__g1798760__2D__d1909756__2D__Reviews__2D__Jackson__5F__Hole__5F__Shooting__5F__Experience__2D__Jackson__5F__Hole__5F__Wyoming__2E__html-Jackson_Hole_Shooting_Experience-Jackson_Hole_Wyoming.html>

<https://www.tripadvisor.com/UserReviewEdit-g1798760-d1909756-e__2F__Attraction__5F__Review__2D__g1798760__2D__d1909756__2D__Reviews__2D__Jackson__5F__Hole__5F__Shooting__5F__Experience__2D__Jackson__5F__Hole__5F__Wyoming__2E__html-Jackson_Hole_Shooting_Experience-Jackson_Hole_Wyoming.html>

**Comment by dscotese** [comment]
Sure, except for the byline :-).  I was about to fix it, but left it to you.  I like the additional graphic too.

I edited the category description to A) have the content of the main How I Became page, and B) Indicate that you're working on adding them.

If you use the link at the bottom of the email to "view it on Github" then you can add a comment without all the extra stuff (see how I edited your comment above).

**Comment by Shepardh1** [comment]
Cool, do I have to learn and use github, or can I just do my thing with
Wordpress... seems less complicated for a dude without a computer
engineer's mind...?

|

*Shepard Humphries' Vision <http://shepardhumphries.com/vision.html> |
Owner |  **www.ShootInJH.com* <http://www.shootinjh.com/>
*Jackson Hole Shooting Experience  |  **Proudly the international leader in
luxury entertainment shooting.*
*    <http://www.linkedin.com/pub/shepard-humphries/14/53/80b>*

<https://www.tripadvisor.com/UserReviewEdit-g1798760-d1909756-e__2F__Attraction__5F__Review__2D__g1798760__2D__d1909756__2D__Reviews__2D__Jackson__5F__Hole__5F__Shooting__5F__Experience__2D__Jackson__5F__Hole__5F__Wyoming__2E__html-Jackson_Hole_Shooting_Experience-Jackson_Hole_Wyoming.html>

<https://www.tripadvisor.com/UserReviewEdit-g1798760-d1909756-e__2F__Attraction__5F__Review__2D__g1798760__2D__d1909756__2D__Reviews__2D__Jackson__5F__Hole__5F__Shooting__5F__Experience__2D__Jackson__5F__Hole__5F__Wyoming__2E__html-Jackson_Hole_Shooting_Experience-Jackson_Hole_Wyoming.html>

<https://www.tripadvisor.com/UserReviewEdit-g1798760-d1909756-e__2F__Attraction__5F__Review__2D__g1798760__2D__d1909756__2D__Reviews__2D__Jackson__5F__Hole__5F__Shooting__5F__Experience__2D__Jackson__5F__Hole__5F__Wyoming__2E__html-Jackson_Hole_Shooting_Experience-Jackson_Hole_Wyoming.html>

<https://www.tripadvisor.com/UserReviewEdit-g1798760-d1909756-e__2F__Attraction__5F__Review__2D__g1798760__2D__d1909756__2D__Reviews__2D__Jackson__5F__Hole__5F__Shooting__5F__Experience__2D__Jackson__5F__Hole__5F__Wyoming__2E__html-Jackson_Hole_Shooting_Experience-Jackson_Hole_Wyoming.html>

**Comment by dscotese** [comment]
Sure, just wordpress is fine.  GH in this case is just so other people can see what's going on.

**Comment by Shepardh1** [comment]
Thanks, now I feel a bit more useful... please let me know if I do anything
outside of your preferences...  :)

Is the plan to transition the main voluntaryist page to wordpress in time?

|

*Shepard Humphries' Vision <http://shepardhumphries.com/vision.html> |
Owner |  **www.ShootInJH.com* <http://www.shootinjh.com/>
*Jackson Hole Shooting Experience  |  **Proudly the international leader in
luxury entertainment shooting.*
*    <http://www.linkedin.com/pub/shepard-humphries/14/53/80b>*

<https://www.tripadvisor.com/UserReviewEdit-g1798760-d1909756-e__2F__Attraction__5F__Review__2D__g1798760__2D__d1909756__2D__Reviews__2D__Jackson__5F__Hole__5F__Shooting__5F__Experience__2D__Jackson__5F__Hole__5F__Wyoming__2E__html-Jackson_Hole_Shooting_Experience-Jackson_Hole_Wyoming.html>

<https://www.tripadvisor.com/UserReviewEdit-g1798760-d1909756-e__2F__Attraction__5F__Review__2D__g1798760__2D__d1909756__2D__Reviews__2D__Jackson__5F__Hole__5F__Shooting__5F__Experience__2D__Jackson__5F__Hole__5F__Wyoming__2E__html-Jackson_Hole_Shooting_Experience-Jackson_Hole_Wyoming.html>

<https://www.tripadvisor.com/UserReviewEdit-g1798760-d1909756-e__2F__Attraction__5F__Review__2D__g1798760__2D__d1909756__2D__Reviews__2D__Jackson__5F__Hole__5F__Shooting__5F__Experience__2D__Jackson__5F__Hole__5F__Wyoming__2E__html-Jackson_Hole_Shooting_Experience-Jackson_Hole_Wyoming.html>

<https://www.tripadvisor.com/UserReviewEdit-g1798760-d1909756-e__2F__Attraction__5F__Review__2D__g1798760__2D__d1909756__2D__Reviews__2D__Jackson__5F__Hole__5F__Shooting__5F__Experience__2D__Jackson__5F__Hole__5F__Wyoming__2E__html-Jackson_Hole_Shooting_Experience-Jackson_Hole_Wyoming.html>

**Comment by dscotese** [comment]
Yes, that's the plan.

**Comment by dooglio** [comment]
You can do both. Make your changes in Wordpress. Tweak the theme, etc.
Then make sure you do regular backups to github so we can mirror the site.

**Comment by frydaho** [comment]
I'm a WP template designer...still need help?

Just got the email; I'll clone the repo when I'm back to my laptop.

**Comment by dscotese** [comment]
Shepard Humphries did all the "How I Became" stories.  I just finished the "Action" section, and now I'm working on the Articles section.

**Comment by dscotese** [comment]
David Butler is working on the Articles section now.  I wrote the following instructions:

1. I have one browser window on the "legacy" page at http://voluntaryist.com/articles/ (You'll notice that this URL gets a #name added to it - this is so that AddThis can track the URL if it's copied and pasted to someone else.)
2. I have another window open on the "WP" page at http://voluntaryist.com/Articles/ (The only difference is the case of the A - since there is no folder on the server named Articles with a capital A, WordPress shows the category page - you can also use http://voluntaryist.com/category/Articles/ and you get the same page).
     The bolded "Issue" entries on this page are categories that are under the parent Articles category.  I wrote some custom code (in /wp/wp-content/themes/devalu and lower folders) that causes them to be displayed that way.
3. I note the first article on the legacy site that is not also on the WP site.
     If it's in an Issue for which there is not yet a category, then I go add that Issue category to WP giving it Articles as the parent category (Dashboard - Posts - Categories).
4. I visit the article on the legacy site and copy the title.
5. I create a new post and paste the title in.
6. I copy the byline and article contents from the legacy site and paste them into the new post.  The byline should be in an h4 tag (WP's "Heading 4" style), and centered.  Sometimes this requires a little finagling.
7. I make sure the right Issue # category is checked on the right (but that can be fixed later) and Publish it.
8. I repeat steps 3 - 7 until I'm bored with it.  Sometimes I read the article :-).

I have been removing the " - Issue Num. ##" from the titles as I move them into WordPress, and Carl said that removing that part of the titles is fine.

**Comment by Shepardh1** [comment]
While it takes additional time, I suggest we be mindful of SEO.  If we do
not already have the "yoast" plugin installed, it would probably be wise.

   1. If possible, find a free & legal picture and add it to the article,
   making sure the article keywords are copy and pasted as the atl tag for the
   photo.
   2. If possible, add the keywords at the bottom of the article to that
   the density is higher.


*Shepard Humphries  **|  **www.ShootInJH.com* <http://www.shootinjh.com/>
*Jackson Hole Shooting Experience  |  **Proudly the international leader in
luxury entertainment shooting.*
*    <http://www.linkedin.com/pub/shepard-humphries/14/53/80b>*
<http://nomadrifleman.com>

**Comment by dscotese** [comment]
Yoast has been installed and Shepard is managing it.

**Comment by dscotese** [comment]
I noticed that the Yoast plugin added a "readability" column to the admin page for the posts, and many of our posts have a "readability" score of "bad."  Let's discuss this to see if there's something we should do while posting to fix that.  I'll make a new issue and copy any emails about it for future reference.

Keep in mind that I LOVE automating stuff, so if we discover that we want to do something to every post, before we start working on that, give me a chance to see how easy it will be for me to do it all in one shot.

**Comment by dscotese** [comment]
David Butler completed the duplication of the articles section.  I think Shepard is working on non-voting.

**Comment by Shepardh1** [comment]
Hello Dave, will you please forward this to all editors?

I (Shepard) made a quick video tutorial about Yoast:
https://youtu.be/e0gr9M1AIwc


*Shepard Humphries *

**Comment by Shepardh1** [comment]
Yes... slowly....  :)


*Shepard Humphries  **|  **www.ShootInJH.com* <http://www.shootinjh.com/>
*Jackson Hole Shooting Experience  |  **Proudly the international leader in
luxury entertainment shooting.*
   * <http://www.linkedin.com/pub/shepard-humphries/14/53/80b> *

**Comment by dscotese** [comment]
CONGRATULATIONS!  We did it!  I don't remember how long ago this task was completed but I wanted to thank everyone who helped.

**Comment by Shepardh1** [comment]
Yay!  At the speed of light we get it done!  lol  :)  It DOES get done and
that is great!  Ya'll rock!


*Shepard Humphries  **|  **www.ShootInJH.com* <http://www.shootinjh.com/>
*Jackson Hole Shooting Experience  |  **Proudly the international leader in
luxury entertainment shooting.*

<https://shootinjh.com/ding/>
